crates/actor/src/lib.rs

```diff
@@ -68,11 +68,30 @@ mod test {
             //);
             // TODO
             #[cfg(not(feature = "module"))]
-            let (_, _, _): (
-                wasmcloud::bus::host::FutureResult,
-                wasi::io::streams::InputStream,
-                wasi::io::streams::OutputStream,
-            ) = wasmcloud::bus::host::call("mycompany:mypackage/interface.operation").unwrap();
+            {
+                let (_, _, _): (
+                    wasmcloud::bus::host::FutureResult,
+                    wasi::io::streams::InputStream,
+                    wasi::io::streams::OutputStream,
+                ) = wasmcloud::bus::host::call(None, "mycompany:mypackage/interface.operation")
+                    .unwrap();
```

Comment:
Flagging because I see `unwrap` but I'm fine with this if it's just temporary code.

Reply:
That's a test and this is never actually called